### PR TITLE
Refactor AWS client

### PR DESCRIPTION
**Pull request by bigkraig**

### CODE REVIEW

**Comment by bigkraig** [comment]
the only place left with the global `Cloudsvc` is in `loadbalancer/main.go` now

File: internal/ingress/metric/main_test.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package metric
```
Comment (M00nF1sh) [inline internal/ingress/metric/main_test.go]: what's this used for?

Comment (bigkraig) [reply internal/ingress/metric/main_test.go]: makes coveralls count the package in our overall test coverage

**Pull request merged**